public/index.html

```diff
@@ -43,17 +43,20 @@
       }
       button {
         font-size: 120%;
-      }
-      @media screen and (max-width: 830px) {
-        button {
-          font-size: 60%;
-        }
+        padding: 16px 30px;
       }
       @media screen and (max-width: 970px) {
-        button {
+        .words-button {
           font-size: 70%;
+          padding: 10px 20px;
         }
       }
+      @media screen and (max-width: 830px) {
```

Comment:
Out of curiousity, why move this section down?

Reply:
Oh, just to have the `max-width`s descending, like the `max-height`s do below.

Reply:
🤦‍♀